pymodm/fields.py

```diff
@@ -1106,7 +1125,8 @@ def to_python(self, value):
                 for item in value]
 
     def to_mongo(self, value):
-        return [doc.to_son() for doc in value]
+        return [EmbeddedDocumentField._value_to_mongo(self.related_model, doc)
+                for doc in value]
```

Comment:
It seems strange to call this method in another class that isn't in this class' hierarchy. See above comment for a suggestion on how to clear this up.